packages/canary-react/src/stories/ic-data-table.stories.mdx

```diff
@@ -1088,36 +1088,38 @@ const DataTable = () => {
 export default DataTable;
 ```
 
-### Links and elements in data
+### Links and slotted elements in data
 
 Custom HTML elements can be slotted or passed via the `data` prop to display in certain cells. When using the slotted method, the slot name follows the format of `{COLUMN_TAG}-{ROW_INDEX}`.
 
 <Canvas withSource="none">
-  <Story name="Links and Elements in data">
+  <Story name="Links and Slotted elements in data">
     <IcDataTable
       caption="Links and Elements in data"
       columns={COLS_ELEMENTS}
       data={DATA_REACT_ELEMENTS}
+      showPagination
     >
       {DATA_REACT_ELEMENTS.map((_, index) => (
         <>
           <IcButton
             key={`actions-${index}`}
             variant="destructive"
             slot={`actions-${index}`}
+            title="Delete row (top level tooltip)"
```

Comment:
I think these changes need reflecting in the "code example" code block below so that the code matches the functionality that is being rendered in this demo